src/ProjectSystemTools/BuildLogging/Model/Frontend/FrontendBuildTableDataSource.cs

```diff
@@ -3,43 +3,40 @@
 using System;
 using System.Collections.Immutable;
 using System.ComponentModel.Composition;
-using Microsoft.Build.Evaluation;
-using Microsoft.Build.Framework;
+using System.Linq;
+using Microsoft.VisualStudio.ProjectSystem.Tools.BuildLogging.Model.RpcContracts;
 using Microsoft.VisualStudio.ProjectSystem.Tools.BuildLogging.UI;
-using Microsoft.VisualStudio.ProjectSystem.Tools.Providers;
 using Microsoft.VisualStudio.Shell.TableManager;
 
-namespace Microsoft.VisualStudio.ProjectSystem.Tools.BuildLogging.Model
+namespace Microsoft.VisualStudio.ProjectSystem.Tools.BuildLogging.Model.FrontEnd
 {
-    [Export(typeof(IBuildTableDataSource))]
-    internal sealed class BuildTableDataSource : ITableEntriesSnapshotFactory, IBuildTableDataSource
+    [Export(typeof(IFrontEndBuildTableDataSource))]
+    internal sealed class FrontEndBuildTableDataSource : ITableEntriesSnapshotFactory, IFrontEndBuildTableDataSource
     {
         private const string BuildDataSourceDisplayName = "Build Data Source";
         private const string BuildTableDataSourceIdentifier = nameof(BuildTableDataSourceIdentifier);
         private const string BuildTableDataSourceSourceTypeIdentifier = nameof(BuildTableDataSourceSourceTypeIdentifier);
 
         private readonly object _gate = new object();
-        private readonly EvaluationLogger _evaluationLogger;
-        private readonly RoslynLogger _roslynLogger;
 
         private ITableManager _manager;
         private ITableDataSink _tableDataSink;
         private BuildTableEntriesSnapshot _lastSnapshot;
-        private ImmutableList<Build> _entries = ImmutableList<Build>.Empty;
+        private ImmutableList<UIBuildSummary> _entries = ImmutableList<UIBuildSummary>.Empty;
+
+        private readonly IBuildLoggerService _loggerService;
 
         public string SourceTypeIdentifier => BuildTableDataSourceSourceTypeIdentifier;
 
         public string Identifier => BuildTableDataSourceIdentifier;
 
         public string DisplayName => BuildDataSourceDisplayName;
 
-        public bool SupportRoslynLogging => _roslynLogger.Supported;
-
-        public bool IsLogging { get; private set; }
+        public bool SupportRoslynLogging { get; }
 
         public int CurrentVersionNumber { get; private set; }
 
-        public ITableManager Manager
+        private ITableManager Manager
```

Comment:
No one uses this property, other than to set it to `null`. It can be removed altogether, along with the `_manager` field.